account/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+class Account(models.Model):
+    email      = models.CharField(max_length = 50)
```

Comment:
지금은 View 로직에서 이메일 중복 처리를 막고있지만, unique=True 로 설정하면 만약 중복된 값이 들어가도 DB단에서 에러가 발생해서 추후에도 디버깅하기가 쉽습니다. 그래서 하기와 같이 설정하는 것 추천드립니다.
```suggestion
    email      = models.CharField(max_length = 50, unique=True)
```